tests/console/commands/plugin/test_add.py

```diff
@@ -161,6 +162,57 @@ def test_add_with_git_constraint_with_extras(
     )
 
 
+@pytest.mark.parametrize(
+    "url",
+    [
+        "git+https://github.com/demo/poetry-plugin2.git#subdirectory=subdir",
+        "git+https://github.com/demo/poetry-plugin2.git@master#subdirectory=subdir",
+    ],
+    ids=["norev", "rev"],
```

Comment:
This could be simplified to be a tuple of `(url, rev)` with `None` for no rev I think -- it would be more in line with our other tests that way.

Reply:
We don't really need `ids` anymore after the test change.